cfsetup.sh

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit
 set -o pipefail
-
+# TODO: convert to .profile
```

Comment:
Is this TODO done by this [this new .profile](https://github.com/GSA/catalog.data.gov/blob/68203482643f9b45eda23157f6f956482a1e890f/.profile)/.

Reply:
Good question: no. We discovered we could use a .profile file to do basically what cfsetup.sh is doing in a custom way, and thought we should do it the "normal" way. See Bret's comments [here](https://github.com/GSA/datagov-deploy/issues/3251#issuecomment-946103785).
The reason I left a TODO instead of doing the work is that I noticed cfsetup.sh uses bash, but .profile uses shell. I decided it was a separate task to merge those together, and I wasn't familiar with what could be done (can we make .profile a bash script? What are the ramifications of this? What's best practice?). I'm going to make a comment in that dashboard ticket requesting to look at this when the ticket is taken on....